PR-table-related CSS improvements.

robinlinden opened the pull request:
Increase font-size in PR-table.
Decrease margins and padding around content.
Center state.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/toktok/website/18)
<!-- Reviewable:end -->

GrayHatter commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18#-:-K_MIGI4RdeaFPUvldZb:bnfp4nl)*
<!-- Sent from Reviewable.io -->

GrayHatter commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18)*
<!-- Sent from Reviewable.io -->

abueide commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18)*
<!-- Sent from Reviewable.io -->

abueide commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18#-:-K_NKAiZ3tDYr85-UaQc:b-jbsf8z)*
<!-- Sent from Reviewable.io -->

Commit:
PR-table-related CSS improvements.
Increase font-size in PR-table.
Decrease margins and padding around content.
Center state.

GrayHatter commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18#-:-Ka-k5wmVdznXcziBDFw:b-jbsf8z)*
<!-- Sent from Reviewable.io -->

iphydf commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18#-:-Ka20un3gMkUlgA4aIIj:b-jbsf8z)*
<!-- Sent from Reviewable.io -->

cebe commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/18#-:-Ka24WnGSlH2dNVU2G6D:b-jbsf8z)*
<!-- Sent from Reviewable.io -->
